recipes/pyScienceMode/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "pysciencemode" %}
+{% set version = "1.1.4" %}
+{% set python_min = "3.10" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://files.pythonhosted.org/packages/51/a4/14e4d9e859a0109c4e44c36f7e759ae8f3d24a603c72ddf28e9ee9919566/pysciencemode-1.1.4.tar.gz
```

```suggestion
  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/pysciencemode-{{ version }}.tar.gz
```

This is a strange URL: https://files.pythonhosted.org/ Why is this being used and not `pypi.org` given that https://pypi.org/project/pysciencemode/ exists? If you generate the recipe with

```
grayskull pypi pysciencemode
```

you'll get 

```yaml
  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/pysciencemode-{{ version }}.tar.gz
```